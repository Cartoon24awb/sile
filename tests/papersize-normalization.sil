\begin[papersize=A-6]{document}
% The paper size being capitalized and hyphenated is non-standard but supported.
% This test makes sure case and hyphen normalization works.
\nofolios
\end{document}
