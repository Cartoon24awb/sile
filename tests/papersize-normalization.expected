Set paper size 	297.6377985	419.5275636
Begin page
End page
Finish
